utilities/decrypt_data/decryption_for_program_service.py

```
@@ -0,0 +1,127 @@
+                                migrate_children(conn, children)
+                            break
+                except Exception as e:
+                    logger.exception("Failed processing parent %s: %s", pid, e)
+
+    logger.info("=== Migration complete ===")
+
+
+if __name__ == "__main__":
+    logger.info("=== STARTING migration for `%s` ===", CONFIG["table"])
+    run_migration()
+    logger.info("=== FINISHED migration for `%s` ===", CONFIG["table"])
```

Flake8 (7.2.0)

[warning] no newline at end of file

(W292)